setup/native-script.txt

```diff
@@ -0,0 +1,13 @@
+
+# A boxter script to setup Windows machine for NativeScript development
+
+# install deps with choco
+
+cinst googlechrome
+cinst nodejs.install
+cinst jdk8
+cinst gradle -version 2.3
+cinst android-sdk
+
+# setup android sdk
+echo yes | cmd /c $env:localappdata\Android\android-sdk\tools\android update sdk --filter tools,platform-tools,android-22,build-tools-22.0.1,sys-img-x86-android-22,extra-android-m2repository,extra-google-m2repository,extra-android-support --all --no-ui
```

Comment:
Just noted that you are installing `android-22`, while we also support `android-23` with `1.4.0` of {N} CLI
But anyway, we can keep the current code until we are sure everything is working fine with `android-23`